decidim-participatory_processes/app/models/decidim/participatory_process.rb

```diff
@@ -43,6 +43,9 @@ class ParticipatoryProcess < ApplicationRecord
 
     has_many :features, as: :participatory_space, dependent: :destroy
 
+    has_many :participatory_process_private_users, class_name: "Decidim::ParticipatoryProcessPrivateUser", foreign_key: "decidim_participatory_process_id", dependent: :destroy
```

Comment:
This looks really similar to the code at Assemblies. What about moving the common logic to a Concern at core and rename the users to `participatory_space_private_users`?